example/dev-env-setup/Vagrantfile

```diff
@@ -0,0 +1,111 @@
+######################
+# Vagrant File Start #
+######################
+
+VAGRANTFILE_API_VERSION = "2"
+
+Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+
+    # RackHD SERVER
+    config.vm.define "dev_ansible" do |target|
+        target.vm.box = "rackhd/rackhd-dev"
+        target.vm.provider "virtualbox" do |v|
+            v.memory = 4096
+            v.cpus = 4
+            v.customize ["modifyvm", :id, "--nicpromisc2", "allow-all", "--ioapic", "on"]
+        end
+
+        # NOTE: virtualbox shared folder syncing can be quite slow, but shared folders
+        # can be configured as NFS mounts to the host instead.
+        # To do so, append: `, type: "nfs"` to all of the target.vm.synced_folder commands
+        # below, then on the host system, run `sudo nfsd start`.
+        # Additionally, uncomment the below configuration:
+
+        # For NFS mounts, a host only network is needed. Feel free to change the IP.
+        # target.vm.network "private_network", ip: "192.168.50.4"
+
+        # Sync optional local workspace directory to the vagrant instances src/ path.
+        # REPO is optional when WORKSPACE is defined, used for synchronizing single RackHD repos
+        # otherwise it assumes all service repos have been cloned and built under the WORKSPACE path.
+        # NOTE: See further below for instructions on how to use an NFS mount for faster synced file performance
+        if ENV['WORKSPACE']
+          if ENV['REPO'] == "on-tasks"
+            target.vm.synced_folder "#{ENV['WORKSPACE']}/#{ENV['REPO']}", "/home/vagrant/src/on-taskgraph/node_modules/on-tasks/"
+            target.vm.synced_folder "#{ENV['WORKSPACE']}/#{ENV['REPO']}", "/home/vagrant/src/on-http/node_modules/on-tasks/"
+          elsif ENV['REPO'] == "on-core"
+            target.vm.synced_folder "#{ENV['WORKSPACE']}/#{ENV['REPO']}", "/home/vagrant/src/on-taskgraph/node_modules/on-core/"
+            target.vm.synced_folder "#{ENV['WORKSPACE']}/#{ENV['REPO']}", "/home/vagrant/src/on-http/node_modules/on-core/"
+            target.vm.synced_folder "#{ENV['WORKSPACE']}/#{ENV['REPO']}", "/home/vagrant/src/on-tftp/node_modules/on-core/"
+            target.vm.synced_folder "#{ENV['WORKSPACE']}/#{ENV['REPO']}", "/home/vagrant/src/on-dhcp-proxy/node_modules/on-core/"
```

Comment:
You should sync on-core first and then on-tasks.
Also I didn't see on-tasks in on-core sync operation.